Unify

suggoitanoshi opened the pull request:
This is basically a merge between two files, `username_api.py` and `frontend.py`. They are merged into a file, `username_checker.py` which contains both the api and the frontend.

andrewda commented on README.md:
```diff
@@ -2,7 +2,7 @@
 [![Join the chat at https://gitter.im/username-availability-checker/Lobby](https://badges.gitter.im/username-availability-checker/Lobby.svg)](https://gitter.im/username-availability-checker/Lobby?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge&utm_content=badge)
 
-Check if a username can or cannot be taken on a bunch of popular social websites (such as Twitter, Instagram, Soundcloud, Github etc.) 
+Check if a username can or cannot be taken on a bunch of popular social websites (such as Twitter, Instagram, Soundcloud, Github etc.)
```
Try to keep unrelated changes to a minimum. If you'd like, you can create a new commit with these spacing changes.

vex8 replied:
Looks like atom automatically removes unneeded spaces at end of every lines. I'll revert those changes and add only necessary changes.

andrewda commented on README.md:
```diff
@@ -37,11 +37,10 @@ copy .env.copy .env
 
 
-### Running the web app 
+### Running the web app
```
^

andrewda commented on username_checker.py:
```diff
@@ -27,7 +44,6 @@
 handler.setFormatter(formatter)
 logger.addHandler(handler)
 
-
```
Unrelated change

andrewda commented on username_checker.py:
```diff
@@ -41,14 +57,56 @@ def filter(self, record):
         u=username
     )
 
-
```
^

andrewda commented on username_checker.py:
```diff
@@ -87,11 +156,21 @@ def get_avatar(website, username):
         return url
     else:
+        log_response(
+            'Not expected result: {}/{}'.format(website, username),
```
"Not expected" -> "Unexpected"

vex8 replied:
Oops, completely missed that. :/

suggoitanoshi committed:
username_checker.py: Unify frontend and backend

This commit merges `frontend.py` and `username_api.py` into
`username_checker.py` and moves API endpoint to `/api/`.

Closes
https://github.com/manu-chroma/username-availability-checker/issues/104

suggoitanoshi committed:
README.md: Reflect unification

This commit updates readme to reflect changes in unification

suggoitanoshi committed:
username_checker.py: Fix whitespaces

This fixes error in coala caused by whitespaces.

suggoitanoshi committed:
test_username_api.py: Update tests

This updates the tests to use `username_checker.py` instead of
`username_api.py`
